crates/walrus-core/src/lib.rs

```diff
@@ -198,6 +198,7 @@ impl Debug for BlobId {
 /// A quilt patch is a individual blob within a quilt.
 #[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize, Hash)]
 #[serde(rename_all = "camelCase")]
+#[cfg_attr(feature = "utoipa", derive(utoipa::ToSchema))]
 pub struct QuiltPatchId {
```

Comment:
Important: We need to explicitly add the schema to the specs in `crates/walrus-service/src/client/daemon/openapi.rs`. In fact, I realized that we are already missing the `ObjectIdSchema` in the aggregator... 😬